homeassistant/components/onvif/__init__.py

```diff
@@ -99,7 +94,7 @@ async def async_populate_options(hass, entry):
     """Populate default options for device."""
     options = {
         CONF_EXTRA_ARGUMENTS: DEFAULT_ARGUMENTS,
-        CONF_RTSP_TRANSPORT: RTSP_TRANS_PROTOCOLS[0],
+        CONF_RTSP_TRANSPORT: next(iter(RTSP_TRANSPORTS)),
```

Comment:
Optional suggestion: Could consider just having a constant DEFAULT_RTSP_TRANSPORT = "tcp" as it may not be obvious from the constant dict that the order matters.

Reply:
This does make it a little more clear, but not sure it is worth defining and importing another constant (assuming we'd be defining it in stream and using it in onvif). What if we just just put a comment where RTSP_TRANSPORTS is defined that the first dict entry should be considered the default?